toolkit/tools/internal/customizationmacros/customizationmacros.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+/*
+customizationmacros package provides functions to add customization macros to a root directory. These macros are saved in the
+root directory's default rpm macros directory. Each customization has a corresponding macro file that is generated as needed.
+*/
+package customizationmacros
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/rpm"
+)
+
+const (
+	// macro files to customize the installer and final images
+	DisableDocsMacroFile         = "macros.installercustomizations_disable_docs"
+	ConfigureRpmLocalesMacroFile = "macros.installercustomizations_customize_locales"
+)
+
+var (
+	// Universal header comments for all customization macro files
+	customizationMacroHeaderComments = []string{
+		"# This macro file was dynamically generated by the Azure Linux Toolkit image generator",
+		"# based on the configuration used at image creation time.",
+		"",
+	}
+	docComments []string = []string{
+		"# This stops documentation files (anything fine in a package which is %%doc) from being installed.",
```

Comment:
Not sure I understood the "anything fine in a package which is %%doc" part. If we're saying that `_excludedocs` removes files marked with `%doc`, then that's also not the full picture - the macro makes RPM skip all files it considers documentation (that includes files under a bunch of hard-coded paths).

Reply:
I'll just say "# This stops anything rpm considers a documentation files from being installed."